### PR TITLE
chore: make clean script clean generated css files

### PR DESCRIPTION
**Pull request by piofinn**
affects: @fremtind/jkl-accordion, @fremtind/jkl-bullet-list, @fremtind/jkl-button,
@fremtind/jkl-card, @fremtind/jkl-checkbox, @fremtind/jkl-core, @fremtind/jkl-datepicker,
@fremtind/jkl-divider-line, @fremtind/jkl-dropdown, @fremtind/jkl-field-group,
@fremtind/jkl-hamburger, @fremtind/jkl-loader, @fremtind/jkl-logo, @fremtind/jkl-message-box,
@fremtind/jkl-radio-button, @fremtind/jkl-text-input, @fremtind/jkl-toggle-switch

## 📥 Proposed changes

The `clean` script only removed build javascript files and types. Update the script in style packages to also remove generated css files.

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: packages/accordion/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     ],
     "license": "MIT",
     "scripts": {
-        "clean": "rimraf build/ dist/ .cache node_modules/",
+        "clean": "rimraf **/*.css",
```
Comment (lfbergee) [inline packages/accordion/package.json]: Isn't parcel in use in most style pkgs? In that case, we should keep the `dist/` remove

Comment (piofinn) [reply packages/accordion/package.json]: Good point! I'll add it back

Comment (piofinn) [reply packages/accordion/package.json]: `node_modules/` should probably also still be in there 🤔

**Comment by piofinn** [comment]
@lfbergee can you take another look?

**Pull request merged**